lib/person.rb

```diff
@@ -0,0 +1,29 @@
+require 'date'
+require 'pry-byebug'
+require './lib/library.rb'
+
+class Person
+
+  attr_accessor :library
+
+  def initialize
+    @library = Library.new
+  end
+
+
+  def check_books
```

Comment:
This method doesn't belong in this class IMO. As a person you go to the library to check for books and not the other way around..